base/dates/io.jl

```diff
@@ -252,13 +253,21 @@ const SLOT_RULE = Dict{Char, Type}(
     's' => Millisecond,
 )
 
-slot_order(::Type{Date}) = (Year, Month, Day)
-slot_order(::Type{DateTime}) = (Year, Month, Day, Hour, Minute, Second, Millisecond)
-
-slot_defaults(::Type{Date}) = map(Int64, (1, 1, 1))
-slot_defaults(::Type{DateTime}) = map(Int64, (1, 1, 1, 0, 0, 0, 0))
+const FORMAT_DEFAULTS = Dict{Type, Any}(
```

Comment:
The `Any` here allows TimeZones.jl to add and return a default timezone I presume?